task_10/Rabbit.css

```diff
@@ -0,0 +1,351 @@
+body{
+    margin: 0;
+}
+@font-face{
+    font-family: BebasNeueRegular;
+    src: url(./font/BebasNeueRegular.ttf);
+}
+@font-face{
+    font-family: StMarie;
+    src: url(./font/StMarie-Thin.otf);
+}
+/* Header */
+header{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    background-color: white;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+}
+.header{
+    display: flex;
+    margin: 0 100px;
+}
+.header div{
+   margin-top: 10px;
+   margin-right: 40px;
+}
+.logo{
+   display: flex;
+   flex-direction: column;
+   justify-content: center;
+}
+.logo div:first-child{
+    color: #fb9c06;
+    font-size: 27px;
+}
+.logo div:last-child{
+    color: #6e6d6b;
+    font-size: 35px;
+    font-family: BebasNeueRegular;
+}
+.menu{
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #9f9c9c;
+    margin: 0 100px;
+}
+.menu-item {
+    height: 48px;
+    width: 55px;
+    font-family: StMarie;
+    margin: 0 15px;
+    background-image: url(./image/sprite.png);
+}
+.home{
+    background-position: 0px 94px;
+}
+.about{
+    background-position: 218px 94px;
+    height: 48px;
+    width: 30px;
+}
+.work{
+    background-position: 730px 94px;
+}
+.blog{
+    background-position: 410px 95px;
+}
+.contact{
+    background-position: 615px 94px;
+    height: 48px;
+    width: 70px;
+} 
+/* Header */
+
+/* section      */
+section{
+    height: 270px;
+    background-image: url(./image/big-orange-bg.png);
+    display: flex;
+    align-items: center;
+    background-size: cover;
+    border-top: 3px solid #fcba50;
+    border-bottom: 3px solid #fcba50;
+    font-family: BebasNeueRegular;
+    margin-top: 110px;
+}
+.name-section{
+     font-size: 35px;
+     height: 40px;
+     line-height: 40px;
+     padding: 5px 10px;
+     margin-left: 100px;
+     text-align: center;  
+}
+ div h3:first-child{
```

Comment:
Please don't this this selector anymore)

.name-section h3:first-child - it is more correct